src/echarts.ts

```diff
@@ -92,6 +92,9 @@ import { getVisualFromData, getItemVisualFromData } from './visual/helper';
 import LabelManager from './label/LabelManager';
 import { deprecateLog } from './util/log';
 import { handleLegacySelectEvents } from './legacy/dataSelectAction';
+// default import ZH and EN lang
+import langEN from "../i18n/langEN";
+import langZH from "../i18n/langZH";
 
```

Comment:
I think it's better to put builtin `langEN` and `langZH` in the `src/i18n`. To keep all the source files in `src` folder. Or it may have unexpected issues when bundling. For examples, typescript compiler will check if the files are in the root `src` folder.